lib/matchers/duration.js

```diff
@@ -6,8 +6,17 @@ var TYPE_ERROR = 'should be a duration string (e.g. \"10s\")';
 module.exports = function (opts) {
 
   if (!opts) opts = {};
-  var minValue = (typeof opts.min === 'string') ? ms(opts.min) : 0;
-  var maxValue = (typeof opts.max === 'string') ? ms(opts.max) : Number.MAX_VALUE;
+
+  var minValue = (opts.min != null) ? ms(opts.min) : 0;
+  var maxValue = (opts.max != null) ? ms(opts.max) : Number.MAX_VALUE;
+
+  if (typeof minValue !== 'number') {
```

Comment:
Yeah, true. I saw that `nullms` weirdness just last week. I guess I'm always mildly, irrationally concerned when I see `typeof`.